projects/canopy/src/lib/card/card.component.spec.ts

```diff
@@ -92,15 +99,41 @@ describe('LgCardComponent', () => {
     });
 
     it('should expect card header to render', () => {
-      expect(
-        fixture.debugElement.query(By.directive(LgCardHeaderComponent)),
-      ).toBeDefined();
+      expect(debugElement.query(By.directive(LgCardHeaderComponent))).toBeTruthy();
     });
 
     it('should expect card content to render', () => {
+      expect(debugElement.query(By.directive(LgCardContentComponent))).toBeTruthy();
+    });
+  });
+
+  describe('when there is lg-card-hero-img and lg-card-content', () => {
+    beforeEach(() => {
+      const localFixture = MockRender(`
+        <lg-card>
+         <lg-card-hero-img imageUrl="/test"></lg-card-hero-img>
+         <lg-card-content>Content</lg-card-content>
+        </lg-card>
+      `);
+
+      debugElement = localFixture.debugElement;
+      el = debugElement.children[0].nativeElement;
+      component = debugElement.children[0].componentInstance;
+      localFixture.detectChanges();
+    });
+
+    it('should expect card hero-image to render', () => {
+      expect(debugElement.query(By.directive(LgCardHeroImageComponent))).toBeTruthy();
```

Comment:
They will display as the order we have in the card. 
<img width="436" alt="Screenshot 2023-08-03 at 13 47 13" src="https://github.com/Legal-and-General/canopy/assets/68600084/c808417f-d06e-426d-bc89-33f475858625">

So for example even if I use it like this :

```
      <lg-card>
        <lg-card-content></lg-card-content>
        <lg-card-hero-img></lg-card-hero-img>
      </lg-card>
```
it will display the img component first. I would not add tests to check this as all the other tests also do not check this and if we add it here, to keep consistency, all of them should have them, so I would say it is not mandatory to test this because it will always render in the correct order.

Reply:
Ah yes good point 👍